x-pack/plugins/index_management/public/application/sections/home/index_list/index_table/index_table.js

```diff
@@ -470,37 +439,69 @@ export class IndexTable extends Component {
     } = this.props;
 
     const { includeHiddenIndices } = this.readURLParams();
+    const hasContent = !indicesLoading && !indicesError;
 
-    let emptyState;
+    if (!hasContent) {
+      const renderNoContent = () => {
+        if (indicesLoading) {
+          return (
```

Comment:
Returning early here again would make the code easier to read.

Reply:
Do you mean converting `else if` to `if`? From my reading of the code, it looks like it's already returning early.

Reply:
Yes, once you have an early return there is no need of `else if`. Just code styling thing, nothing important.

```js
if (something) {
  return true;
}
return false;

// instead of

if (something) {
  return true;
} else {
  return false;
}
```